src/main/java/com/amazonaws/encryptionsdk/internal/DecryptionHandler.java

```diff
@@ -61,7 +60,8 @@
 
     private CryptoHandler contentCryptoHandler_;
 
-    private DataKey<K> dataKey_;
+    private SecretKey dataKey_;
```

Comment:
s/dataKey_/plaintextDataKey_

Reply:
This class is going to revert to the original now that I am using an independent EncryptionMaterials and DecryptionMaterials